pom.xml

```diff
@@ -449,7 +506,11 @@
                     <doUpdate>false</doUpdate>
                     <revisionOnScmFailure>${maven.build.timestamp}</revisionOnScmFailure>
                 </configuration>
-            </plugin> 
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
```

Comment:
Yes, because we need to get coverage for the code we write. The code coverage report will be generated when the tests are executed. Therefore it's very important to know the coverage and to understand whether the tests have covered critical code sections.